apps/dashboard/src/lib/components/Apps/index.svelte

```diff
@@ -80,10 +82,18 @@
 
     <div class="mb-2 relative {getAppClass(APPS_CONSTANTS.APPS.COMMENTS, $apps.selectedApp)}">
       <IconButton
-        toolTipProps={{ title: 'Comments', hotkeys: ['Ctrl/Command', '1'] }}
+        toolTipProps={{
+          title: `${
+            $globalStore.isOrgSite && !$currentOrg.customization.apps.comments
+              ? $t('course.navItem.lessons.disabled')
+              : $t('course.navItem.lessons.comments.title')
+          }`,
+          hotkeys: ['Ctrl/Command', '1']
+        }}
         value={APPS_CONSTANTS.APPS.COMMENTS}
         onClick={handleAppClick}
         buttonClassName="relative"
+        disabled={$globalStore.isOrgSite && !$currentOrg.customization.apps.comments}
```

Comment:
While testing I realized that there was no custom styling for how the disabled behavior of an icon button looked like.

In future PR, I would expect you to notice such thing and either ask or recommend something